src/App.tsx

```diff
@@ -1,73 +1,109 @@
-import React from 'react';
+import { useState, useRef } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import debounce from 'lodash.debounce';
+import { Autocomplete } from './components/autocomplete';
+import { NoMatch } from './components/nomatch';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const initialPerson: Person = {
+    name: '',
+    born: 0,
+    died: 0,
+    slug: '',
+    fatherName: '',
+    motherName: '',
+    sex: 'm',
+  };
+
+  const [selectedPerson, setSelectedPerson] = useState<Person>(initialPerson);
```

Comment:
```suggestion
  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
```

and then this below:

```
const { name, born, died } = selectedPerson || {};
```
Because actually, there is no person selected initially